VocalRaidAssistant.lua

```diff
@@ -32,7 +32,6 @@ if locales[L] then
 	C_Timer.After(30, function() addon:prettyPrint(msg) end)
 end
 
-local tostring = tostring
 local pairs = pairs
 local GetTime = GetTime
```

Comment:
Can you remove this line? This was used here before, but is unused now. It is being used in Sound.lua.

Reply:
This line is removed o0
We can remove GetTime here, too